joblibspark/backend.py

```diff
@@ -202,11 +207,16 @@ def mapper_fn(_):
         except ImportError:
             pass
 
+        if SafeFunction is None:
+            return self._get_pool().apply_async(
+                run_on_worker_and_fetch_result, callback=callback
```

Comment:
I think now we should use `_wrap_func_call` instead of `SafeFunction` ?

see https://github.com/joblib/joblib/blob/b9be27d7ce405417572fee6fcd9317c6b6ff67b6/joblib/_parallel_backends.py#L283